war/src/main/less/abstracts/theme.less

```diff
@@ -70,11 +70,11 @@
   // Header dimensions
   --header-item-border-radius: 4px;
 
-  // Breadcrumbs and footer
-  --breadcrumbs-bg: #f8f8f8;
```

Comment:
The removed variables have two usages in the dark theme and in the solarized theme plugin.

Objections about removing or adjusting those in the plugins?

Reply:
I've updated the dark-theme plugin https://github.com/jenkinsci/dark-theme-plugin/pull/219

Will take a look at Solarized now..

Reply:
Thanks 👍🏻

Reply:
Solarized update is here: https://github.com/jenkinsci/solarized-theme-plugin/pull/60